src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -45,16 +92,48 @@ public void parse_validOneArg_returnsFindCommand() throws ParseException {
     @Test
     public void parse_validMultipleArgs_returnsFindCommand() throws ParseException {
```

Comment:
Perhaps can also consider test cases that has unnecessary parenthesis for a more comprehensive testing? E.g.
`find (n/Alice && t/friends)`

Reply:
Oh I think I see what you might have meant -- maybe have specific cases where parentheses do or don't alter precedence? Added in 20249368